app/pages/EditBreakingNewsPage.tsx

```diff
@@ -46,29 +53,60 @@ export const EditBreakingNewsPage = () => {
     ]);
   };
 
-  const onSave = (articleId: string) => {
-    const article = breakingNewsArticles.find(
-      ({ id }: any) => id === articleId
+  const articleStatus = (article: NewsArticle): JSX.Element | null => {
```

Comment:
Related to my other comment, to turn this into a true React component, change the argument from just a plain article to an an object with an `article` property. Also, the convention for React components is to capitalize the first letter of the name, since React components were originally written only as classes, before they added function-based components.

```suggestion
  const ArticleStatus = ({ article}: { article: NewsArticle}) => {
```

(The return type can usually be omitted from React components as well)